server/context.go

```diff
@@ -37,6 +37,7 @@ import (
 // Context defaults.
 const (
 	defaultAddr               = ":26257"
+	defaultPGPort             = "15432"
```

Comment:
We shouldn't add this without making it configurable via a flag, since a hard-coded value makes it impossible to run multiple servers on the same machine, or to run the tests while a local server is running.